exo-go/src/service_config_helpers/index_test.go

```diff
@@ -21,6 +21,32 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 })
 
+var _ = Describe("GetServiceData", func() {
+
+	It("should not return an error when application.yml is valid", func() {
```

Comment:
GetServiceData never returns an error so I think this test description needs to be updated to actually describe what this function is doing. Maybe `joins the public and private services into a single map`